server/src/main/java/org/apache/druid/client/cache/MemcachedCacheConfig.java

```diff
@@ -63,6 +63,12 @@ public class MemcachedCacheConfig
   @JsonProperty
   private String locator = "consistent";
 
+  @JsonProperty
+  private boolean sslconnection = false;
+
+  @JsonProperty
+  private boolean autodiscovery = false;
```

Comment:
let's use `autoDiscovery` here to keep in line with existing camel-case naming conventions used for other configs